src/components/ThirdPartyCollectionDetailPage/CollectionPublishButton/CollectionPublishButton.tsx

```diff
@@ -53,11 +53,30 @@ const CollectionPublishButton = (props: Props) => {
     onClick(collection.id, itemIds, buttonAction)
   }, [collection, items, buttonAction])
 
+  const itemsTryingToPublish = useMemo(
+    () => items.filter(item => !itemCurations.find(itemCuration => itemCuration.itemId === item.id)).length,
```

Comment:
I think it's not necessary to check the status here. I think that in order to get the ones that are gonna be published, we need the items with NO curations at all, because if they are pending/accepted/rejected, the action would be the push changes one.